tests/dragonfly/utility.py

```diff
@@ -46,12 +46,8 @@ async def wait_available_async(client: aioredis.Redis, timeout=10):
     start = time.time()
     while (time.time() - start) < timeout:
         try:
-            await client.get("key")
+            await client.ping()
```

Comment:
why did you change this?
we want here to make sure that server exits loading stage